src/firefly/js/visualize/saga/ActiveRowToImageWatcher.js

```diff
@@ -20,53 +22,63 @@ import CoordinateSys from '../CoordSys';
 
 
 
+
 function willFitOnScreenAtCurrentZoom(pv) {
     const {w,h}=computeBoundingBoxInDeviceCoordsForPlot(primePlot(pv)) ?? {};
     return (pv.viewDim.width+3>=w && pv.viewDim.height+3>=h);
 }
 
+const getTableActions= () =>  [TABLE_LOADED, TABLE_SELECT, TABLE_HIGHLIGHT, TABLE_UPDATE, TBL_RESULTS_ACTIVE, TABLE_REMOVE];
+const getImageActions= () =>  [ImagePlotCntlr.CHANGE_ACTIVE_PLOT_VIEW, ImagePlotCntlr.ANY_REPLOT];
```

Comment:
not sure why these are defined as functions and not just variables?

Reply:
because of cross import issues